test/lib/rule-finder.js

```diff
@@ -77,57 +85,107 @@ describe('rule-finder', function() {
 
   it('specifiedFile (relative path) is passed to the constructor', function() {
     var ruleFinder = getRuleFinder(specifiedFileRelative)
-    assert.deepEqual(ruleFinder.getUnusedRules(), ['baz-rule', 'react/bar-rule', 'react/baz-rule', 'react/foo-rule'])
+    assert.deepEqual(ruleFinder.getUnusedRules(), [
+      'baz-rule',
+      'react/bar-rule',
+      'react/baz-rule',
+      'react/foo-rule',
+      'something/two-rule',
```

Comment:
As I understand the [ESLint docs](http://eslint.org/docs/developer-guide/working-with-plugins), the rules are to be referenced by their containing plugin's name (regardless of their possibly scoped package's name).

So for `eslint-plugin-<plugin-name>`'s rules, as well as `@scope/eslint-plugin-<plugin-name>`, its rules should be configured via `<plugin-name>/<rule-name>`.

Please correct me, if I'm wrong.